source/run-command/tests/unified/run-command.yml

```diff
@@ -0,0 +1,242 @@
+description: runCommand spec requirements
+
+schemaVersion: "1.5"
+
+createEntities:
+  - client:
+      id: client
+      observeEvents: [commandStartedEvent]
+  - database:
+      id: plainDb
+      client: client
+      databaseName: plainDb
+  - collection:
+      id: collection
+      database: plainDb
+      collectionName: collection
+  - database:
+      id: dbWithRC
+      client: client
+      databaseName: dbWithRC
+      databaseOptions:
+        readConcern:
+          level: 'local'
+  - database:
+      id: dbWithWC
+      client: client
+      databaseName: dbWithWC
+      databaseOptions:
+        writeConcern:
+          w: 0
+  - session:
+      id: session
+      client: client
+  # Stable API test
+  - client:
+      id: stableApiClient
+      observeEvents: [commandStartedEvent]
+      serverApi:
+        version: "1"
+        strict: true
+  - database:
+      id: stableApiDb
+      client: stableApiClient
+      databaseName: stableApiDb
+
+initialData:
+- collectionName: collection
+  databaseName: plainDb
+  documents: [ { _id: 0 } ]
+
+tests:
+  - description: executes a ping with $db attached
```

Comment:
It would be better to be more specific with the test case names to communicate both what the expected behavior is and what condition prompts it, e.g., in this case, `always attaches the $db to the ping command` (although, should it be to "every" command?)

Reply:
> "always attaches $db to the given command"

Yes it will always be present on every command, even meta commands like hello target the `admin.$cmd` namespace.